addons/account/models/account_payment.py

```diff
@@ -746,6 +788,33 @@ def _synchronize_to_moves(self, changed_fields):
                 'line_ids': line_ids_commands,
             })
 
+    def _create_paired_internal_transfer_payment(self):
```

Comment:
We should synchronize both transfer payments, not only create it

Reply:
This has been discused and it was decide to only create it.

Reply:
Could you remind me the arguments that lead to that conclusion?

Reply:
Dixit the pad, under "# TESTS (...)  TSB 2021/03/24 : "
> Considered premature optimization and bug-prone to cover this ; discussed between TSB/JBW ; JBW/LAS